src/main/java/com/google/sps/model/GmailClient.java

```diff
@@ -70,6 +71,17 @@
     }
   }
 
+  /**
+   * Given some search queries, combine them into a larger query Will not remove
```

Comment:
Nit: I think you are missing a period before "Will".

Reply:
Oops - It was originally a new line, but spotless removed that. All fixed!